app/views/newspaper_works/base/_attribute_rows.html.erb

```diff
@@ -0,0 +1,36 @@
+<%# newspaper_core_metadata attributes %>
+<%= presenter.attribute_to_html(:date_modified, label: t('hyrax.base.show.last_modified'), html_dl: true) %>
+<%= presenter.attribute_to_html(:alternative_title, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:genre, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:place_of_publication_label, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:issn, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:lccn, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:oclcnum, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:held_by, render_as: :faceted, html_dl: true) %>
+<%# newspaper_article attributes %>
+<%= presenter.attribute_to_html(:author, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:photographer, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:volume, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:issue_number, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:geographic_coverage, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:extent, render_as: :faceted, html_dl: true) %>
+<%= presenter.attribute_to_html(:publication_date, render_as: :faceted, html_dl: true) %>
+<%# newspaper_page attributes %>
```

Comment:
Missing `section` and `page_number` attributes from NewspaperPage model here. Both of those should be rendered as plain text, not as faceted links.